src/sfdt/blocksProcess.ts

```diff
@@ -58,27 +81,27 @@ export const processBlock = (block: any, callbackInline: TCallback, callbackBloc
 //  - characterFormat
 export const processBlocks = (parent: any, callback: TCallback) => {
 	if (!parent.sections) {
-		// console.warn('Missing: sections', parent)
-		return false
+		console.warn('Missing: sections', parent);
```

Comment:
can you please comment out this line? We dont want debug log in prod :)

Reply:
done